enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -0,0 +1,231 @@
+for OpenShift on ppc64le verification. This would help diversify our CI effort.
+- A non-x86 IPI installer would serve as a first step towards improving the ease of deployments on non-x86 hadware.
+- Power hardware is especially targeted to AI and secure workloads, so building out cloud capabilities will help expand
+our workload footprint.
+- IPI is generally easier to install and set up quickly for demos and proof of concepts, when can help lower the barrier
+of entry for power optimized workloads.
+
+### Goals
+
+- Provide a way to install OpenShift on Power VS infrastructure using the OpenShift installer with IPI provisioned
+against IBM Cloud APIs.
+- Implement the cluster-api provider to provide scaling and managing the cluster nodes post install.
+
+### Non-Goals
+
+- UPI support. We've already documented how to install on pp64le hardware using UPI.
+
+## Proposal
+
+In order to leverage the Power VS platform, we will first need to implement an IBM Cloud Provider. This provider will
+interface with IBM Cloud APIs and ultimately enable the OpenShift Installer to provision VM resources using Power VS,
+that will be used as worker and masters of the cluster. Unlike some of the existing providers, this depends on image
+availability of CoreOS in the PowerVS cloud the OVA format. There are currently 2 proposals of how that can be
+achieved.
+
+The expected workflow would be as follows:
+1. The installer is invoked with credentials for ones IBM Cloud environment.
+2. Once invoked, the installer interacts with a terraform provider to create a new Power VS Service Instance.
+3. The installer will then import a CoreOS OVA boot image from an existing cloud store to the service instance.
+4. Once imported the installer can start building the private network for the nodes.
+5. The bootstrap and master nodes will be created on in the private network. These are called Power VS instances.
+6. In order to expose this private network and provide load balancing, the IBM Cloud APIs will leverage the IBM
+Cloud [Direct Link][direct-link-website] offering. This will allow for both the ingress load balancer and the cluster
+nodes to talk over the private network on a high-speed link.
+7. Bootstrapping completes as normal, and workers are added.
+
+### User Stories
+
+- A user of this installer would like to see a cluster running within an hour of kicking off the installer.
+- The expectation is that this is fully automated.
+
+### Implementation Details/Notes/Constraints
+
+#### CoreOS OVA Considerations
+There were two main implementation ideas for making the CoreOS OVA boot image available to the Virtual Server Instance.
+
+1. Import the Image from a Cloud Store
+This is the implementation we believe is best, since it works most similarly to other cloud environments. In this setup,
+the CoreOS build pipeline would publish an OVA image to a cloud store (e.g. AWS bucket). Then, we will use the regular
```

Comment:
The cloud store example will be IBM Cloud Object Storage (COS) bucket. That's the only place from where it can be imported